libnavui-androidauto/src/main/java/com/mapbox/androidauto/map/MapboxCarMapLoader.kt

```diff
@@ -65,6 +65,17 @@ class MapboxCarMapLoader : MapboxCarMapObserver {
         mapboxMap = null
     }
 
+    /**
+     * Returns the current style contract. If an override has not been set the default is returned.
+     */
+    fun getCurrentExtension(isDarkMode: Boolean): StyleContract.StyleExtension {
```

Comment:
I think `getCurrentStyle` was fine. The same way as in Maps SDK the function is named `loadStyle`, not `loadExtension`.